uicomponent-compose/main/src/main/java/io/github/droidkaigi/feeder/AppContent.kt

```diff
@@ -46,7 +48,13 @@ fun AppContent(
         drawerState = drawerState,
         drawerShape = MaterialTheme.shapes.large.copy(all = CornerSize(0.dp)),
         drawerContent = {
-            DrawerContent { route ->
+            val navBackStackEntry by navController.currentBackStackEntryAsState()
```

Comment:
How about this implementation? 👀 

```kotlin
 var currentDestination by remember { mutableStateOf<NavDestination>()}
    navController.addOnDestinationChangedListener { controller, destination: NavDestination, arguments ->
        currentDestination = destination
    }
```

Reply:
I will check it a little more. 👀

Reply:
If you do this implementation (addOnDestinationChangedListener), a lot of listeners will be added, so you have to remember or do something. 👀